everyvoice/tests/test_wizard.py

```diff
@@ -755,9 +755,9 @@ def test_wrong_fileformat_festival(self):
         with patch_menu_prompt((0, 1, 2, 3), multi=True) as stdout:
             format_step.run()
         output = stdout.getvalue().replace(" \n", " ")
-        self.assertRegex(output, "does not look like a .*'psv'")
-        self.assertRegex(output, "does not look like a .*'tsv'")
-        self.assertRegex(output, "does not look like a .*'csv'")
+        self.assertRegex(output, r"does not look like a .*'psv'".replace(" ", r"\s"))
+        self.assertRegex(output, r"does not look like a .*'tsv'".replace(" ", r"\s"))
+        self.assertRegex(output, r"does not look like a .*'csv'".replace(" ", r"\s"))
```

Comment:
These failed due to my specific screen size and working path (there was a newline between "does not" and "look like"), so I made them more robust.
Not related to this PR, I know, but I had to do it to run the unit tests.